src/abstractdataframe/io.jl

```diff
@@ -165,7 +167,7 @@ function latex_escape(cell::AbstractString)
     replace(cell, ['\\','~','#','$','%','&','_','^','{','}']=>latex_char_escape)
 end
 
-function Base.show(io::IO, ::MIME"text/latex", df::AbstractDataFrame)
+function Base.show(io::IO, ::MIME"text/latex", df::AbstractDataFrame; summary=true)
```

Comment:
Sorry, I thought we printed the summary in this method too. Since we don't, there's no point in taking that argument.